client/src/views/BallotTrackerStart.vue

```diff
@@ -148,8 +148,7 @@ onMounted(() => {
 
 <style type="text/css" scoped>
 .BallotTrackerStart {
-  font-family: "Open Sans";
```

Comment:
Was the font not supposed to be "Open Sans", or is it set elsewhere?

Reply:
It is now globally declared for the whole `body` so there's no need to have it in each class. I'm cleaning those up when I found them throughout the components I'm currently working on.